tests/unit/acquisition/test_utils.py

```diff
@@ -41,7 +42,7 @@
     ],
 )
 def test_split_acquisition_function(
-    f: AcquisitionFunction, x: np.ndarray, split_size: int, expected_batches: int
+    f: AcquisitionFunction, x: np.ndarray[Any, Any], split_size: int, expected_batches: int
```

Comment:
so here we explicitly state the type of objects in arrays, similar as with tuple, List etc?

Reply:
Yes: the parameters refer to shape and dtype (though shape isn't properly supported at the moment, and we currently don't handle dtypes well enough to use that part). There's a cleaner shorthand in numpy.typing called `npt.NDArray` but that doens't work with older versions of mypy.